packages/table/src/common/grid.ts

```diff
@@ -308,10 +308,18 @@ export class Grid {
         const cardinality = Regions.getRegionCardinality(region);
         switch (cardinality) {
             case RegionCardinality.CELLS: {
-                const cellRect0 = this.getCellRect(region.rows[0], region.cols[0]);
-                const cellRect1 = this.getCellRect(region.rows[1], region.cols[1]);
-                const offsetLeft = region.cols[0] === 0 ? 0 : 1;
-                const offsetTop = region.rows[0] === 0 ? 0 : 1;
+                const [rowStart, rowEnd] = region.rows;
+                const [colStart, colEnd] = region.cols;
+
+                // if the region is outside the bounds of the table, don't display it
+                if (this.isGhostIndex(rowStart, colStart)
+                 || this.isGhostIndex(rowEnd, colEnd)) {
```

Comment:
awkward indentation; always whole multiple of four spaces (half expect this to fail linter, but i guess it didn't).

```
if (A
    || B) {
}